### PR TITLE
Update db

### PR DESCRIPTION
**Pull request by nicolas-heigvd**
All your bases are belong to us...

### CODE REVIEW

**Comment by nicolas-heigvd** [comment]
@verdonarthur Fixed ?

File: packages/backend/src/migration/1603879411988-initDomains.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        await queryRunner.query(`CREATE DOMAIN modos.uint2 AS SMALLINT CHECK(VALUE >= 0 AND VALUE < 65535);`, undefined);
+        await queryRunner.query(`CREATE DOMAIN modos.angle AS FLOAT CHECK(VALUE >= 0 AND VALUE < 360);`, undefined);
+        await queryRunner.query(`CREATE DOMAIN modos.cam_make AS TEXT CHECK(VALUE IN ('apple','canon','dji','casio','fujifilm','garmin','google','gopro','hasselblad','hewlett-packard','htc','huawei','jvc','kodak','konica','leica','lg','logitech','microsoft','minolta','motorola','nikon','nintendo','nokia','olympus','panasonic','parrot','pentax','polaroid','ricoh','rollei','samsung','sanyo','seiko-epson','sigma','sony','sony-ericsson','toshiba','vodafone','xiaomi','other'));`, undefined);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        throw new Error("This migration cannot be reversed.");
+    }
+
+}
```
Comment (verdonarthur) [inline packages/backend/src/migration/1603879411988-initDomains.ts:16]: missing line jump

File: packages/backend/src/migration/1603887777590-initSpatialRel.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        await queryRunner.query(`CREATE TABLE IF NOT EXISTS modos."nodes"(id INTEGER GENERATED ALWAYS AS IDENTITY, lat FLOAT NOT NULL, lon FLOAT NOT NULL, osmid BIGINT UNIQUE NOT NULL, highway VARCHAR, data_source text NOT NULL DEFAULT 'osm', geom geometry(Point, 4326) NOT NULL, uuid UUID UNIQUE NOT NULL, CONSTRAINT nodes_pkey PRIMARY KEY (id), CONSTRAINT check_source CHECK (data_source='osm' OR data_source='other'));`, undefined);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        throw new Error("This migration cannot be reversed.");
+    }
+
+}
```
Comment (verdonarthur) [inline packages/backend/src/migration/1603887777590-initSpatialRel.ts:14]: Missing line jump

File: packages/backend/src/migration/1603887777592-alterTables.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE modos."observation" ADD COLUMN eid INT REFERENCES modos.edges(id);`, undefined);
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        throw new Error("This migration cannot be reversed.");
+    }
+
+}
```
Comment (verdonarthur) [inline packages/backend/src/migration/1603887777592-alterTables.ts:13]: missing line jump

**Review by verdonarthur — CHANGES_REQUESTED** [review]
Fix the line jump and see with @AlphaHydrae to resolve the other problem with osmdata schema

**Comment by nicolas-heigvd** [comment]
I hope it can be merged now, everything is schema agnostic; it's a big mix in the public schema by default.

Be careful; the postgis spatial reference table is nearby the application tables. Do not touch it!

**Review by verdonarthur — APPROVED** [review]
Not sure to understand all the modification you are doing but it's ok for me

**Pull request merged**